holmes/plugins/toolsets/infrainsights/mongodb_toolset.py

```
@@ -0,0 +1,524 @@
+        return f"Get MongoDB server status for instance: {instance_name}"
+
+
+class InfraInsightsMongoDBToolset(BaseInfraInsightsToolset):
+    """MongoDB toolset for InfraInsights"""
+    
+    def __init__(self, config: Optional[Dict[str, Any]] = None):
+        super().__init__(config)
+        
+        self.name = "InfraInsights MongoDB"
+        self.description = "Tools for investigating MongoDB databases, collections, and documents in InfraInsights"
+        self.tags = [ToolsetTag.CLUSTER]
```

Ruff (0.12.2)

Undefined name `ToolsetTag`

(F821)